pkg/ciliumenvoyconfig/exp_controller.go

```diff
@@ -42,24 +41,31 @@ type cecControllerParams struct {
 	Metrics        experimentalMetrics
 
 	NodeLabels     *nodeLabels
```

Comment:
`LocalNodeStore`, `NodeLabels` & `Log` are unused.

Reply:
Removed and double-checked all fields that they're used.